src/components/FixtureRow.tsx

```diff
@@ -54,8 +52,10 @@ function FixtureRow({fixture}: FixtureRowProps) {
             </div>
           </div>
           <div className="d-flex flex-column align-items-start">
-            <div>{date.toDateString()}</div>
-            <div>{formattedTime(date)}</div>
+            {/* Sat Dec 23 2023 */}
+            <div>{format(date, 'EEE MMM d yyyy', {locale: localesMap[navigator.language]})}</div>
```

Comment:
@astone123  Can we encapsulate this logic to "get the user's current locale.Locale" somewhere so that we don't have to define `localsMap` and do `localesMap[navigator.language]}` every time we want it?